concourse/pipelines/deploy.yml

```diff
@@ -327,7 +327,10 @@ jobs:
     - get: govuk-infrastructure
       trigger: true
     - file: govuk-infrastructure/concourse/pipelines/deploy.yml
-      set_pipeline: deploy-apps-test
+      set_pipeline: ((pipeline_name))
```

Comment:
I think one can use `self` here. It might be deprecated in future, but should be good for now.

Reply:
Yeah, I can do that